src/tests/mediaPresenter.test.ts

```diff
@@ -0,0 +1,96 @@
+import { MediaPresenter } from "../MediaPresenter";
+describe("media presenter", () => {
+  let presenter: MediaPresenter;
+  const imageFile = new File(["(⌐□_□)"], "blank.png", { type: "image/png" });
+  const videoFile = new File(["(⌐□_□)"], "blank.mp4", { type: "video/mp4" });
+  const textFile = new File(["(⌐□_□)"], "blank.pdf", {
+    type: "application/pdf",
+  });
+  beforeEach(() => {
+    presenter = new MediaPresenter();
+  });
+  describe("addFile", () => {
+    it("should append a file element to the files list", () => {
+      expect(presenter.filesLength).toEqual(0);
+      presenter.addFile(imageFile);
+      expect(presenter.filesLength).toEqual(1);
+      expect(presenter.media[0].file).toEqual(imageFile);
+    });
+
+    it("should return false if file is not an image or a video", () => {
+      let returnVal = presenter.addFile(textFile);
```

Comment:
nit: `const`, and same below

In JS, we use `let` only if a value is going to be reassigned, whereas `const` is for immutable values